### PR TITLE
[operator-trivy][docs] Add info about scanning namespace `default` by default

### PR DESCRIPTION
**Pull request by unreturned**
## Description
More usable label to copy and paste.
Clarify info about `default` namespace: we have env var here
https://github.com/deckhouse/deckhouse/blob/fecf4c44b0c2a50ebf40fdcb5915a671742ff9b2/ee/modules/500-operator-trivy/templates/deployment.yaml#L70-L71
and the reason, why it's done explained here https://github.com/deckhouse/deckhouse/pull/5662

## Why do we need it, and what problem does it solve?
Improve documentation.

## Why do we need it in the patch release (if we do)?
Not necessarily.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: operator-trivy
type: chore
summary: Add info about scanning namespace `default` by default
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @unreturned. I've started the workflow for label 'deploy/web/test'...

**Review by mortelumina98 — APPROVED** [review]

File: ee/modules/500-operator-trivy/docs/README_RU.md [hunk 1]
```diff
@@ -5,4 +5,13 @@ description: operator-trivy — модуль Deckhouse для периодиче
 
 Модуль позволяет запускать периодическое сканирование на уязвимости. Базируется на проекте [Trivy](https://github.com/aquasecurity/trivy).
 
-Сканирование осуществляется каждые 24 часа в пространствах имен с лейблом `security-scanning.deckhouse.io/enabled`.
+Модуль выполняет сканирование каждые 24 часа в пространствах имён, которые содержат метку `security-scanning.deckhouse.io/enabled=""`.
```
Comment (Zhbert) [inline ee/modules/500-operator-trivy/docs/README_RU.md:8]:
```suggestion
Модуль каждые 24 часа выполняет сканирование в пространствах имён, которые содержат метку `security-scanning.deckhouse.io/enabled=""`.
```

File: ee/modules/500-operator-trivy/docs/README_RU.md [hunk 2]
```diff
@@ -5,4 +5,13 @@ description: operator-trivy — модуль Deckhouse для периодиче
...
+Если в кластере отсутствуют пространства имён с указанной меткой, сканируется пространство имён `default`.
+
+Как только в кластере обнаруживается пространство имён с меткой `security-scanning.deckhouse.io/enabled=""`, сканирование пространства имён `default` прекращается.
+
+Чтобы снова включить сканирование для пространства имён `default`, необходимо установить метку командой:
+
+```shell
+`kubectl label namespace default security-scanning.deckhouse.io/enabled=""`
```
Comment (Zhbert) [inline ee/modules/500-operator-trivy/docs/README_RU.md:16]:
```suggestion
kubectl label namespace default security-scanning.deckhouse.io/enabled=""
```

**Review by Zhbert — COMMENTED** [review]
Two small fixes.

File: ee/modules/500-operator-trivy/docs/README_RU.md [hunk 3]
```diff
@@ -5,4 +5,13 @@ description: operator-trivy — модуль Deckhouse для периодиче
 
 Модуль позволяет запускать периодическое сканирование на уязвимости. Базируется на проекте [Trivy](https://github.com/aquasecurity/trivy).
 
-Сканирование осуществляется каждые 24 часа в пространствах имен с лейблом `security-scanning.deckhouse.io/enabled`.
+Модуль каждые 24 часа выполняет сканирование в пространствах имён, которые содержат метку `security-scanning.deckhouse.io/enabled=""`.
+Если в кластере отсутствуют пространства имён с указанной меткой, сканируется пространство имён `default`.
+
+Как только в кластере обнаруживается пространство имён с меткой `security-scanning.deckhouse.io/enabled=""`, сканирование пространства имён `default` прекращается.
+
+Чтобы снова включить сканирование для пространства имён `default`, необходимо установить метку командой:
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/README_RU.md:13]:
```suggestion
Чтобы снова включить сканирование для пространства имён `default`, необходимо установить у него метку командой:
```

File: ee/modules/500-operator-trivy/docs/README.md [hunk 4]
```diff
@@ -3,6 +3,15 @@ title: "The operator-trivy module"
...
-The module allows you to run periodic vulnerability scans. The module uses the [Trivy](https://github.com/aquasecurity/trivy) project. 
+The module allows you to run periodic vulnerability scans. The module uses the [Trivy](https://github.com/aquasecurity/trivy) project.
 
-Scanning is performed every 24 hours in namespaces with the `security-scanning.deckhouse.io/enabled ` label.
+Scanning is performed every 24 hours in namespaces that contain the label `security-scanning.deckhouse.io/enabled=""`.
+If there are no namespaces with this label in the cluster, the `default` namespace is scanned.
+
+Once a namespace with the label `security-scanning.deckhouse.io/enabled=""` is detected in the cluster, scanning of the `default` namespace stops.
+
+To re-enable scanning for the `default` namespace, the following label must be applied using this command:
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/README.md:13]:
```suggestion
To re-enable scanning for the `default` namespace, use the following command to set the label to the namespace:
```

File: ee/modules/500-operator-trivy/docs/README_RU.md [hunk 5]
```diff
@@ -5,4 +5,13 @@ description: operator-trivy — модуль Deckhouse для периодиче
 
 Модуль позволяет запускать периодическое сканирование на уязвимости. Базируется на проекте [Trivy](https://github.com/aquasecurity/trivy).
 
-Сканирование осуществляется каждые 24 часа в пространствах имен с лейблом `security-scanning.deckhouse.io/enabled`.
+Модуль каждые 24 часа выполняет сканирование в пространствах имён, которые содержат метку `security-scanning.deckhouse.io/enabled=""`.
+Если в кластере отсутствуют пространства имён с указанной меткой, сканируется пространство имён `default`.
+
+Как только в кластере обнаруживается пространство имён с меткой `security-scanning.deckhouse.io/enabled=""`, сканирование пространства имён `default` прекращается.
+
+Чтобы снова включить сканирование для пространства имён `default`, необходимо установить у него метку командой:
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/README_RU.md:13]:
```suggestion
Как только в кластере обнаруживается пространство имён с меткой `security-scanning.deckhouse.io/enabled=""`, сканирование пространства имён `default` прекращается.
Чтобы снова включить сканирование для пространства имён `default`, необходимо установить у него метку командой:
```

File: ee/modules/500-operator-trivy/docs/README.md [hunk 6]
```diff
@@ -3,6 +3,15 @@ title: "The operator-trivy module"
...
-The module allows you to run periodic vulnerability scans. The module uses the [Trivy](https://github.com/aquasecurity/trivy) project. 
+The module allows you to run periodic vulnerability scans. The module uses the [Trivy](https://github.com/aquasecurity/trivy) project.
 
-Scanning is performed every 24 hours in namespaces with the `security-scanning.deckhouse.io/enabled ` label.
+Scanning is performed every 24 hours in namespaces that contain the label `security-scanning.deckhouse.io/enabled=""`.
+If there are no namespaces with this label in the cluster, the `default` namespace is scanned.
+
+Once a namespace with the label `security-scanning.deckhouse.io/enabled=""` is detected in the cluster, scanning of the `default` namespace stops.
+
+To re-enable scanning for the `default` namespace, use the following command to set the label to the namespace:
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/README.md:13]:
```suggestion
Once a namespace with the label `security-scanning.deckhouse.io/enabled=""` is detected in the cluster, scanning of the `default` namespace stops.
To re-enable scanning for the `default` namespace, use the following command to set the label to the namespace:
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11014](https://github.com/deckhouse/deckhouse/pull/11014) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Pull request merged**